<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>character_creation_window</class>
 <widget class="QDialog" name="character_creation_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Character Creation</string>
  </property>
  <widget class="QLineEdit" name="character_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Character Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="character_type">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Player</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NPC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of Character</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ini-Bonus</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="character_value">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-999</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="save_character">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TP</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="character_life">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create new Character</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
